pyproject.toml

```diff
@@ -23,6 +23,9 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+aws_sdk = [
```

Comment:
I am not sure about the name of the extra. What about just "aws" or "boto3"? Do you know about the naming used in other open source projects?

Reply:
Pydantic AI uses `bedrock`, [source](https://ai.pydantic.dev/models/#bedrock).

Reply:
That is also a possibility...

Reply:
Yeah! Make more sense and I renamed to `bedrock`